evergreen.yml

```diff
@@ -1574,7 +1574,7 @@ buildvariants:
       - name: windows-test-integration-group
       - name: windows-test-result-set-group
       - name: sign
-        run_on: linux-64-amzn-build
+        run_on: ubuntu2204-large
```

Comment:
I added this because `windows-64-vs2019-large` doesn't have docker or Podman; however, I know that `ubuntu2204-large` has docker.

Reply:
Sounds good to me. We absolutely don't have to use a Windows host to sign on Windows, you are 100% correct 👍